web/src/pages/salesforce/salesforce.scss

```diff
@@ -0,0 +1,145 @@
+// -----------------------------------------------------------------------------
+@import "../../styles/variables.scss";
+@import "../../styles/mixins"; 
+// -----------------------------------------------------------------------------
+
+#header {
+    .header-section {
+        background-image: url("https://www.awesomeinc.org/assets/img/aincu/adults/intro-to-web-hero.png");
```

Comment:
@Sisay-su This needs to reference a file in our codebase because this URL will change when we launch the site